src/test/app/FeeVote_test.cpp

```diff
@@ -19,11 +19,203 @@
 
 #include <test/jtx.h>
 
+#include <xrpld/app/ledger/Ledger.h>
+#include <xrpld/app/misc/FeeVote.h>
+#include <xrpld/app/tx/apply.h>
+
 #include <xrpl/basics/BasicConfig.h>
+#include <xrpl/ledger/View.h>
+#include <xrpl/protocol/Feature.h>
+#include <xrpl/protocol/Indexes.h>
+#include <xrpl/protocol/PublicKey.h>
+#include <xrpl/protocol/STTx.h>
+#include <xrpl/protocol/SecretKey.h>
 
 namespace ripple {
 namespace test {
 
+struct FeeSettingsFields
+{
+    std::optional<std::uint64_t> baseFee = std::nullopt;
```

Comment:
std::optional<std::uint64_t> baseFee; // will be initialized to nullopt by its constructor